ember-cli-build.js

```diff
@@ -8,6 +8,17 @@ module.exports = function (defaults) {
     // Add options here
   });
 
+  if ('@embroider/webpack' in app.dependencies()) {
```

Comment:
I'm wondering if it would be better to use a `USE_EMBROIDER` environment variable for this and unconditionally add the dev deps to package.json, instead of using ember-try scenarios. That would allow dependabot to update them, and would not immediately break our CI if embroider releases a version that is no longer compatible.

Reply:
Done!